packages/reports/addon/components/filter-values/date.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Usage:
+ *   {{filter-values/date
+ *       filter=filter
+ *       request=request
+ *       onUpdateFilter=(action 'update')
+ *   }}
+ */
+import Component from '@ember/component';
+import Moment from 'moment';
+import layout from '../../templates/components/filter-values/date';
+import { computed, get } from '@ember/object';
+
+export default Component.extend({
+  layout,
+
+  /**
+   * @property {Array} classNames
+   */
+  classNames: ['date'],
+
+  /**
+   * @property {Moment} selectedDate
+   */
+  selectedDate: null,
+
+  /**
+   * @property {String} displayedDate
+   */
+  displayedDate: computed('filter.values', function() {
+    let filterDate = get(this, 'filter.values.firstObject');
+
+    return filterDate ? Moment(filterDate).format('MMM DD, YYYY') : 'Select date';
+  }),
+
+  actions: {
+    /**
+     * @action setDate
+     * @param {Date} date - new date to set in filter
+     */
+    setDate(date) {
+      this.attrs.onUpdateFilter({
+        values: [Moment(date).format('YYYY-MM-DD')]
```

Comment:
Is the difference between the display format and this intentional?

Reply:
Display format is literally only for display whereas this string is what's passed to the querybuilder and used in the filter param